structs.go

```diff
@@ -890,7 +890,7 @@ type WebhookParams struct {
 	Username  string          `json:"username,omitempty"`
 	AvatarURL string          `json:"avatar_url,omitempty"`
 	TTS       bool            `json:"tts,omitempty"`
-	File      string          `json:"file,omitempty"`
+	File      *File           `json:"-"`
```

Comment:
This should probably be a slice of files rather than a single file to mirror `MessageSend`, and send all of the files in the request similar to [`ChannelMessageSendComplex`](https://github.com/bwmarrin/discordgo/blob/b3cc83b10cce9e09fd126fd9bca044be3f97a4f6/restapi.go#L1554-L1571).